Gemfile

```diff
@@ -48,6 +48,9 @@ gem 'devise'
 # Omniauth
 gem 'omniauth-github'
 
+# Turbolinks form errors compatibility
+gem 'turbolinks_render'
```

Comment:
This is needed to let turbolinks support this piece of code:

```
  def create
    @onboarding = UserOnboardingForm.new(
      user_onboarding_params.merge(user: current_user)
    )

    if @onboarding.save
      redirect_to after_sign_in_path_for(current_user)
    else
      render :show
    end
  end
```

Without the gem, the `render` method won't work.